azure-pipelines.yml

```diff
@@ -43,6 +43,10 @@ stages:
 - stage: build
   displayName: Build
   jobs:
+  - template: /eng/common/templates/job/onelocbuild.yml
+    parameters:
+      CreatePr: false
```

Comment:
So when you make a change to the resx files (and thus to the xlf files they drive) if this setting is true a PR against this repo would automatically be created, which the localization team would party on until everything was localized. We have it false for now for 2 reasons. 1) The first step in onboarding tells you to definitely leave it off. 2) It would normally accomplish that via interacting with a mirrored repo, which we don't have.